typescript/README.md

```diff
@@ -0,0 +1,52 @@
+# TypeScript Examples
+
+![Language-Support: Stable](https://img.shields.io/badge/language--support-stable-success.svg?style=for-the-badge)
+
+This section contains all the CDK code examples written in Typescript. For more information on using the CDK in Typescript, please see the [Developer Guide](https://docs.aws.amazon.com/cdk/latest/guide/work-with-cdk-typescript.html).
+
+## Running Examples
+
+To run a Typescript example, execute the following:
+
+```
+$ npm install -g aws-cdk
+$ cd typescript/EXAMPLE_DIRECTORY
+$ npm install
+$ cdk deploy
```

Comment:
I don't know when `npx ts-node` started being used to run TypeScript apps, but if any example is older than that, it should be updated. Note that there was a bug that caused apps not to be re-transpiled after changes to the TypeScript source code, which could throw off customers if they want to make changes to an example. I made a PR to fix that but it should be added to the existing examples. https://github.com/aws/aws-cdk/pull/11470